src/main/kotlin/no/nav/syfo/database/Database.kt

```diff
@@ -18,7 +18,7 @@ data class DbConfig(
     val password: String,
     val username: String,
     val databaseName: String,
-    val poolSize: Int = 2,
+    val poolSize: Int = 4,
```

Comment:
Siden vi har to Kafka-konsumenterer gående tenker jeg det er fornuftig å utvide poolSize her litt slik at api-kallene også slipper til.